pageserver/benches/bench_walredo.rs

```diff
@@ -26,9 +27,9 @@ fn redo_scenarios(c: &mut Criterion) {
 
     let conf = PageServerConf::dummy_conf(repo_dir.path().to_path_buf());
     let conf = Box::leak(Box::new(conf));
-    let tenant_id = TenantId::generate();
+    let tenant_shard_id = TenantShardId::unsharded(TenantId::generate());
```

Comment:
It will be nice to explain what does `TenantShardId::unsharded` mean.
I looks in shard.rs at the sources of this method and still the meaning of this name is obscure to me...

Reply:
I've tacked that onto the list of things to clean up in the names/docs of helpers https://github.com/neondatabase/neon/issues/6072